### ISSUE TITLE
Attenuation synthesis problem

### ISSUE THREAD

**Issue by Zhentao-Liu** [Ruyi-Zha/naf_cbct#1]
It's a good idea to apply implicit function and hash table to reconstruction CBCT volume. However, there is a problem in attenuation synthesis. In your paper, you give the following equation:
![att](https://user-images.githubusercontent.com/81148025/191879338-42089859-591b-44bf-b642-8f466ed65557.PNG)
While in the code your provide: acc = torch.sum((raw[..., 0] + noise) * dists, dim=-1). I think they are not the same. Would you give some explanation?

**Comment by Ruyi-Zha** [comment]
Thanks for your interest. There is a preprocessing step for raw X-ray images before we actually use them. We do logarithm operation on raw data such that the intensity becomes a simple sum, which is what the code shows.

**Comment by puresky123** [comment]
Hello, may I ask when the raw X-ray images was preprocessed? Has the raw X-ray images in the raw folder you provided been preprocessed

**Comment by Ruyi-Zha** [comment]
Hi, if I remember correctly, the `img.mat` files in the `raw` folder are not preprocessed (normalized), while `dataGenerator/generateData.py` will preprocess (normalize) the 3D volume and pack it into the `*.pickle` file.

**Comment by puresky123** [comment]
Thanks a lot
